.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  root: true,
+  extends: ["universe/native"],
+  ignorePatterns: ["plugin/build"],
```

Comment:
@KiwiKilian, I checked out the branch, expo-app and react-native-app build. I am able to run the tests, build the plugin and generate the docs, but `yarn lint:eslint` hangs for me in the android/build directory. If I change the inorePatterns to just `["build"]` lint works even if I have build directories.  Can you confirm and resolve this issue?

Otherwise, this is good to go.

Reply:
From where did you run `yarn lint:eslint`, the root directory I guess?

I don't have a `./android/build` directory. I only have `./android/rctmln/build` – but that's not generating any problems when running `yarn lint:eslint` from the root directory. How would a generate the `./android/build` directory?

Reply:
Sorry, `packages/react-native-app/android/build`.  You need to run the `packages/react-native-app` example.  cd into the directory and run `yarn android` and it will build for the simulator, creating that build directory.  Afterwards I ran `yarn lint:eslist --debug` to see where it was hanging.

Reply:
Still can't reproduce – works fine for me in any combination. I'm not opposed towards change the `ignorePatterns`, we probably never want to lint on a `build` directory. Just wanting to understand the issue. Does it happen when you delete those build directories and retry afterwards? Maybe it's something about the state of you directories?

Reply:
I've pushed the change anyway.

Reply:
This runs for me now. Thanks.